### ISSUE TITLE
Dcmgr: Improve api response document

### ISSUE THREAD

**Issue by hansode** [axsh/wakame-vdc#600]
[WIP]

### Problem

When launching a new instance, API will responds detail document.
But some launching situations need uuid only.

Current response:

+ `C`: kind of `R`
+ `R`: detail
+ `U`: almost simple
+ `D`: almost simple

**Comment by Metallion** [comment]
Let me just sum up the issue in layman's terms. Basically POST and PUT calls to the API currently provide a lot of information and it's not user friendly. Users are generically only interested in the resource uuid or the fields they've just changed. With the current implementation, they have to scroll a lot to get the correct information on screen. This can be especially annoying when logged in through VNC.

The idea is to only have http GET calls display a full description of a resource. The other calls should have their response trimmed down significantly.
